tests/ui/specs/RangePicker/RangePicker-specs.js

```diff
@@ -8,19 +8,277 @@ describe('RangePicker', function () {
+					await rangePicker.decrementer(rangePicker.self).click();
+					await browser.pause(500);
+					const newValue = await extractValue(rangePicker);
+					expect(newValue).toBe(-1);
+				});
+			});
+
+			describe('wrap', function () {
+				const rangePicker = Page.components.rangePickerWrap;
+
+				it('should increase the value when incrementing the range picker', async function () {
+					const oldValue = await extractValue(rangePicker);
+					expect(oldValue).toBe(0);
+					await rangePicker.incrementer(rangePicker.self).click();
+					await browser.pause(500);
+					const newValue = await extractValue(rangePicker);
+
+					expect(newValue).toBe(1);
+				});
+
+				it('should decrease the value when decrementing the range picker', async function () {
+					const oldValue = await extractValue(rangePicker);
+					expect(oldValue).toBe(0);
+					await rangePicker.decrementer(rangePicker.self).click();
+					expect(await rangePicker.decrementer(rangePicker.self).isFocused()).toBe(true);
+					await browser.pause(500);
+					const newValue = extractValue(rangePicker);
+
+					expect(await newValue).toBe(10);
+				});
+			});
+		});
+
+		describe('horizontal', function () {
+			describe('default', function () {
+				const rangePicker = Page.components.rangePickerHorizontalDefault;
+
+				describe('5-way', function () {
+					it('should change the value forward when incrementing the range picker', async function () {
+						// 5-way down to increment button of horizontal RangePicker
+						for (let i = 0; i <= 7; i++) {
+							await Page.spotlightDown();
+						}
+						await Page.spotlightSelect();
+						await browser.pause(500);
+						const newValue = await extractValue(rangePicker);
+
+						expect(newValue).toBe(5);
+					});
+
+					it('should change the value backward when decrementing the range picker', async function () {
+						// 5-way down to increment button of horizontal RangePicker
+						for (let i = 0; i <= 7; i++) {
+							await Page.spotlightDown();
+						}
+						await Page.spotlightSelect();
+						await browser.pause(500);
+						await Page.spotlightLeft();
+						await Page.spotlightSelect();
+						await browser.pause(500);
+						const newValue = await extractValue(rangePicker);
+
+						expect(newValue).toBe(0);
+					});
+				});
+
+				describe('pointer', function () {
+					it('should increase the value when incrementing the range picker', async function () {
+						await rangePicker.incrementer(rangePicker.self).click();
+						await browser.pause(500);
+						const newValue = await extractValue(rangePicker);
+
+						expect(newValue).toBe(5);
+					});
+
+					it('should decrease the value when decrementing the range picker', async function () {
+						await rangePicker.incrementer(rangePicker.self).click();
+						expect(await rangePicker.incrementer(rangePicker.self).isFocused()).toBe(true);
+						await rangePicker.decrementer(rangePicker.self).click();
+						await browser.pause(500);
+
```

Comment:
Please remove empty line and also be consistent with leaving or not leaving an empty line before expect. (in the other files as well)

Reply:
I deleted all empty spaces